PWGHF/HFL/Tasks/taskSingleMuon.cxx

```diff
@@ -133,12 +133,14 @@ struct HfTaskSingleMuon {
       const auto charge(muon.sign());
       const auto chi2(muon.chi2MatchMCHMFT());
 
-      auto trkMFT = muon.template matchMFTTrack_as<MFTTracksExtra>();
-      if (reduceAmbMft && trkMFT.has_ambMftTrack()) {
-        continue;
-      }
-      if (reduceOrphMft && (!reduceAmbMft) && trkMFT.has_ambMftTrack()) {
-        continue;
+      if (muon.matchMFTTrackId() > 0) {
```

Comment:
I am not sure if you can use the following syntax, but if possible it would be preferred. Could you please try to see if it works?
```suggestion
      if (muon.has_matchMFTTrack()) {
```